src/index.html

```diff
@@ -1,13 +1,63 @@
 <!doctype html>
-<html lang="en">
+<html class="page" lang="en">
```

Comment:
Why adding classes to `.html`?

That is valid, but a little bit weird (you always have a single `html` and don't reuse `.page` class)

If you really need `.page` container - better add it inside `body`

Reply:
In the lessons related to the layout of the landing page, Misha made the classes .page for the html tag and .page__body for the body tag, so I decided that this would be the right solution and tried to apply it in practice. I will make the fix as you told, but I want to know which solution will be the most valid.